mods/ra/maps/soviet-08b/soviet08b.lua

```diff
@@ -0,0 +1,176 @@
+--[[
+   Copyright 2007-2020 The OpenRA Developers (see AUTHORS)
+   This file is part of OpenRA, which is free software. It is made
+   available to you under the terms of the GNU General Public License
+   as published by the Free Software Foundation, either version 3 of
+   the License, or (at your option) any later version. For more
+   information, see COPYING.
+]]
+alliedScouts = { Jeep1, Jeep2, Rifle1, Rifle2, Rifle3, Rifle4 }
```

Comment:
Style nit: Global variables should start with a capital letter. (Applies to several in this file.)